public/app/core/utils/fetch.ts

```diff
@@ -74,7 +74,10 @@ export const isContentTypeApplicationJson = (headers: Headers) => {
   }
 
   const contentType = headers.get('content-type');
-  if (contentType && contentType.toLowerCase() === 'application/json') {
+  if (
+    contentType &&
+    (contentType.toLowerCase() === 'application/json' || contentType.toLowerCase() === 'application/json-patch+json')
```

Comment:
Extending what counts as true for this function will be necessary for PATCH operations in the API platform, which require this content type to be set.